modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/ScalaLagomServerCodegen.java

```diff
@@ -163,6 +164,42 @@ public String toOperationId(String operationId) {
         return camelize(operationId, true);
     }
 
+    private CodegenModel codegenModel(Object o) {
```

Comment:
This helper function may be confusing. It's extracting the top-level model from our object model but reads as if it intends to extract a model from any input object.

Renaming to `extractTopLevelModel` _might_ make it clearer.